### ISSUE TITLE
Pascal VOC mIOU should ignore `void` labels

### ISSUE THREAD

**Issue by bermanmaxim** [martinkersner/train-DeepLab#19]
Hello,
At training, the `void` label in Pascal VOC is mapped to the background class `0`. However, for evaluation purposes, it would make sense to ignore pixels corresponding `void` pixels in the ground-truth, as this is standard for Pascal VOC evaluation. If I'm not mistaken, this step was not done here. I opened a similar issue on [this tensorflow implementation](https://github.com/DrSleep/tensorflow-deeplab-resnet/issues/10#issuecomment-273761232).

**Comment by Hutai** [comment]
@bermanmaxim I really find this problem. How do you update the mIOU code？

**Comment by IssamLaradji** [comment]
You can just zero the prediction mask at the locations where the void pixels are before computing the mIoU.
